### PR TITLE
Lesson 15 turcu alexandru

### PR DESCRIPTION
**Pull request by Munteanu-Alexandru**
adding homework for unittest

### CODE REVIEW

File: lesson_15/lesson_15_Turcu_Alexandru/unittest.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import unittest
+from lesson_15.lesson_15_Turcu_Alexandru import homework
+
+
+class TestSquareArea(unittest.TestCase):
+    def test_area(self):
+        elem = 2
+        area = elem * elem
```
Comment (Liitle) [inline lesson_15/lesson_15_Turcu_Alexandru/unittest.py]: the test idea is good, but you need to make sure you use the code for testing. This test will never fail. You need to use the square class to compute the area


`square = Square(elem,elem,elem,elem)
self.assertEqual(square.area(), area)
`

File: lesson_15/lesson_15_Turcu_Alexandru/unittest.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        self.assertAlmostEqual(area, elem*elem)
+
+    def test_square_type(self):
+        test_homework1 = homework.Polygons(1, 2, 3)
+        self.assertIsInstance(1, int)
```
Comment (Liitle) [inline lesson_15/lesson_15_Turcu_Alexandru/unittest.py]: same as above, this test will never fail, 1 will always be an int. use the code an test on test_homework1.sides[0]

**Review by Liitle — CHANGES_REQUESTED** [review]
Nice work, please make sure you use the code in every test. You need to make the assert on a value generated from code and an expected value

File: lesson_15/lesson_15_Turcu_Alexandru/unittest.py (hunk 2 above)
Comment (Munteanu-Alexandru) [reply lesson_15/lesson_15_Turcu_Alexandru/unittest.py]: So basicly like this , I understand now !  self.assertIsInstance(test_homework1.sides[0], int)

Comment (Liitle) [reply lesson_15/lesson_15_Turcu_Alexandru/unittest.py]: Yes, that is correct, can you also make the changes? The following tests need some changes 

- `test_area` you are not using the code  in test
- `test_square_type` you are not using the code in test
- `test_polygon_perimeter` you are not using the code perimeter should be equal to test_homework2.perimeter()
- `test_sides` not using code, you make the assert on sides with you defined in the test, you need to make the assert on the result that the code gives you

**Review by Liitle — CHANGES_REQUESTED** [review]

**Review by Liitle — APPROVED** [review]

**Pull request merged**